src/renderer/i18n/fr/bonds.ts

```
@@ -17,7 +17,9 @@ const bonds: BondsMessages = {
   'bonds.node.enterMessage': 'Entrez le nœud à surveiller',
   'bonds.validations.nodeAlreadyAdded': 'Le nœud est déjà ajouté',
   'bonds.node.removeMessage': 'Êtes-vous sûr de vouloir supprimer le nœud {node} ?',
-  'bonds.validations.bondStatusActive': "La déliaison d'un nœud actif n'est pas autorisée"
+  'bonds.validations.bondStatusActive': "La déliaison d'un nœud actif n'est pas autorisée",
+  'bonds.tooltip.removeFromWatchlist': 'Retirer ce fournisseur d\'obligations de la liste de surveillance',
```

ESLint

[error] Replace `'Retirer·ce·fournisseur·d\'obligations·de·la·liste·de·surveillance'` with `"Retirer·ce·fournisseur·d'obligations·de·la·liste·de·surveillance"`

(prettier/prettier)